modules/apps/portal-search/portal-search-ranking-web/src/main/resources/META-INF/resources/js/components/ResultsRankingForm.es.js

```diff
@@ -444,30 +432,47 @@ class ResultsRankingForm extends Component {
 			}
 		);
 
-		const {resultIdsHidden} = this.state;
-
 		const {companyId, namespace} = this.context;
 
 		return fetchDocuments(
 			this.props.fetchDocumentsHiddenUrl,
 			{
 				[`${namespace}companyId`]: companyId,
-				[`${namespace}from`]: resultIdsHidden.length,
+				[`${namespace}from`]: DELTA * this.state.hiddenCur,
 				[`${namespace}keywords`]: this.props.searchTerm,
 				[`${namespace}size`]: DELTA
 			}
 		).then(
 			({items, total}) => {
-				const newItems = items ?
-					items.filter(({id}) => !this._initialResultIdsHidden.includes(id)) :
-					[];
 
-				const mappedData = items ? resultsDataToMap(newItems) : {};
+				const definedItems = items ? items : {};
+
+				// Record duplicate results in order to set addedResult property to
```

Comment:
Can we use "get" instead of "record"? "Record" seems like the value is being saved to someplace. 

```suggestion
				// Get duplicate results in order to set addedResult property to
```